### DISCUSSION TITLE
Future timeseries of radiation and sea level pressure

### DISCUSSION THREAD

**Discussion by mullerrobert799** [Deltares/Wflow.jl#286] [Q&A] [answered]
Hello,

I am going to simulate the future period and have temperature and precipitation files of continuous data for future period. But I don't know how to get the radiation and sea level pressure data for future years. Could you please advise me on that? 

Do you have any provision directly in WFLOW/hydroMT where we can either get the future data or maybe make it by using some script or function?

**Comment by verseve** [comment, marked as answer]
Wflow does not provide any data, including forecasting data, or functionality to preprocess this data. You could check for example the Global Forecast System (GFS) from NOAA, data from ECMWF and data from the Deutscher Wetterdienst (ICON model).
